CRM/CiviAwards/Setup/AddAwardsCgExtendsOptionValue.php

```diff
@@ -19,6 +20,8 @@ public function apply() {
     ]);
 
     if ($result['count'] > 0) {
+      $this->toggle(TRUE);
```

Comment:
`$result['count'] > 0` means that the option value does not exist at all. 
No need to toggle this.